modules/gui/vlayer.lua

```diff
@@ -10,8 +10,20 @@ local Event = require 'utils.event' --- @dep utils.event
 local format_number = require('util').format_number --- @dep util
 local config = require 'config.vlayer' --- @dep config.vlayer
 local vlayer = require 'modules.control.vlayer'
+local Selection = require 'modules.control.selection' --- @dep modules.control.selection
+local SelectionConvertArea = 'ConvertArea'
```

Comment:
This must be a unique string. Currently it conflicts with waterfill.
```suggestion
local SelectionConvertArea = 'VlayerConvertChest'
```